framework/doc/content/source/meshgenerators/AdvancedExtruderGenerator.md

```diff
@@ -0,0 +1,39 @@
+# AdvancedExtruderGenerator
```

Comment:
you ll want to do a git mv on  to keep the edit history

Reply:
I tried. It still says `create/delete` instead of `rename`.